spec/README.md

```diff
@@ -0,0 +1,78 @@
+# Ostracon Spec
+
+This is a markdown specification of the Ostracon. Ostracon is designed to have many compatibilities with CometBFT. As such, the Ostracon spec is based on the [CometBFT v0.34.x Spec](https://github.com/cometbft/cometbft/tree/v0.34.x/spec). The Ostracon also has many unique improvements to improve security and performance. This specification clarifies the differences between CometBFT and defines the base data structures, how they are validated, and how they are communicated over the network.
+
+## Contents
+
+> **Note**
+> ![oc](./static/oc.svg) represents items with Ostracon-specific improvements added. It also describes the major changes.
+
+> **Note**
+> ![tm](./static/tm.svg) represents items with specifications equivalent to CometBFT. As such, the article links directly to CometBFT's repository.
```

Comment:
I tried various patterns. I select pattern 1 because I think the difference between ostracon and cometBFT is obvious. How about you?

pattern 1
- [![oc](https://github.com/ulbqb/ostracon/raw/feat/spec/spec/static/oc.svg)Data Structure](./core/data_structures.md): Ostacon uses a more secure proposer election algorithm. Add definitions for the data structures required for this election algorithm.
- [![tm](https://github.com/ulbqb/ostracon/raw/feat/spec/spec/static/tm.svg)Encoding](https://github.com/cometbft/cometbft/blob/v0.34.x/spec/core/encoding.md)
- [![tm](https://github.com/ulbqb/ostracon/raw/feat/spec/spec/static/tm.svg)Genesis](https://github.com/cometbft/cometbft/blob/v0.34.x/spec/core/genesis.md)
- [![oc](https://github.com/ulbqb/ostracon/raw/feat/spec/spec/static/oc.svg)State](./core/state.md): Ostacon uses a more secure proposer election algorithm. Add the data required for this election algorithm to the state.

pattern 2
- [Data Structure](./core/data_structures.md): Ostacon uses a more secure proposer election algorithm. Add definitions for the data structures required for this election algorithm.
- [![tm](https://github.com/ulbqb/ostracon/raw/feat/spec/spec/static/tm.svg)Encoding](https://github.com/cometbft/cometbft/blob/v0.34.x/spec/core/encoding.md)
- [![tm](https://github.com/ulbqb/ostracon/raw/feat/spec/spec/static/tm.svg)Genesis](https://github.com/cometbft/cometbft/blob/v0.34.x/spec/core/genesis.md)
- [State](./core/state.md): Ostacon uses a more secure proposer election algorithm. Add the data required for this election algorithm to the state.

pattern 3
- [Data Structure![oc](https://github.com/ulbqb/ostracon/raw/feat/spec/spec/static/oc.svg)](./core/data_structures.md): Ostacon uses a more secure proposer election algorithm. Add definitions for the data structures required for this election algorithm.
- [Encoding![tm](https://github.com/ulbqb/ostracon/raw/feat/spec/spec/static/tm.svg)](https://github.com/cometbft/cometbft/blob/v0.34.x/spec/core/encoding.md)
- [Genesis![tm](https://github.com/ulbqb/ostracon/raw/feat/spec/spec/static/tm.svg)](https://github.com/cometbft/cometbft/blob/v0.34.x/spec/core/genesis.md)
- [State![oc](https://github.com/ulbqb/ostracon/raw/feat/spec/spec/static/oc.svg)](./core/state.md): Ostacon uses a more secure proposer election algorithm. Add the data required for this election algorithm to the state.

pattern 4
- [Data Structure](./core/data_structures.md): Ostacon uses a more secure proposer election algorithm. Add definitions for the data structures required for this election algorithm.
- [Encoding![tm](https://github.com/ulbqb/ostracon/raw/feat/spec/spec/static/tm.svg)](https://github.com/cometbft/cometbft/blob/v0.34.x/spec/core/encoding.md)
- [Genesis![tm](https://github.com/ulbqb/ostracon/raw/feat/spec/spec/static/tm.svg)](https://github.com/cometbft/cometbft/blob/v0.34.x/spec/core/genesis.md)
- [State](./core/state.md): Ostacon uses a more secure proposer election algorithm. Add the data required for this election algorithm to the state.

Reply:
Of these, pattern 4 is my preference, but we can make a decision at a later revision, especially as no one else seems to have an opinion.